test/src/test/java/org/corfudb/integration/LogReplicationDataReceiverIT.java

```diff
@@ -0,0 +1,305 @@
+            if (i < numTableStandby) {
+                Table<StringKey, IntValueTag, Metadata> tableStandby =
+                        corfuStoreStandby.openTable(NAMESPACE, mapName, StringKey.class,
+                                IntValueTag.class, Metadata.class,
+                                TableOptions.fromProtoSchema(IntValueTag.class, TableOptions.builder().build()));
+                mapNameToMapStandby.put(mapName, tableStandby);
+            } else {
+                Table<StringKey, IntValueTag, Metadata> tableStandby =
+                        corfuStoreStandby.openTable(NAMESPACE, mapName, StringKey.class,
+                                IntValueTag.class, Metadata.class,
+                                TableOptions.fromProtoSchema(NotFederatedTag.class, TableOptions.builder().build()));
+                if (i == numTableStandby) {
+                    mapNameToMapStandby.put(mapName, tableStandby);
+                }
+            }
+        }
+
+        // Open table003 and table004 on standby with different metadata
+        Map<TableName, Table> standbyOnlyTables = new HashMap<>();
+        for (int i = 3; i <= 4; i++) {
+            String mapName = TABLE_PREFIX + i;
+            TableName tableName = TableName.newBuilder()
+                    .setTableName(mapName)
+                    .setNamespace(NAMESPACE)
+                    .build();
+
+            standbyOnlyTables.put(tableName, mapNameToMapStandby.get(mapName));
+
+            CorfuRecord<TableDescriptors, TableMetadata> corfuRecord =
+                    standbyRuntime.getTableRegistry().getRegistryTable().get(tableName);
+            Assert.assertFalse(corfuRecord.getMetadata().getTableOptions().getIsFederated());
+            Assert.assertTrue(corfuRecord.getMetadata().getTableOptions().getRequiresBackupSupport());
+        }
+
+        // Add Data for Snapshot Sync
+        writeToActive(0, NUM_WRITES);
+        // Confirm data does exist on Active Cluster
+        for(Table<StringKey, IntValueTag, Metadata> map : mapNameToMapActive.values()) {
+            assertThat(map.count()).isEqualTo(NUM_WRITES);
+        }
+
+        // Confirm data does not exist on Standby Cluster
+        for(Table<StringKey, IntValueTag, Metadata> map : mapNameToMapStandby.values()) {
+            assertThat(map.count()).isEqualTo(0);
+        }
+
+        pluginConfigFilePath = TEST_PLUGIN_CONFIG_PATH_ACTIVE;
+        startActiveLogReplicator();
+
+        pluginConfigFilePath = TEST_PLUGIN_CONFIG_PATH_STANDBY;
+        startStandbyLogReplicator();
+
+        verifyDataOnStandby(NUM_WRITES);
+
+        for (Map.Entry<TableName, Table> entry : standbyOnlyTables.entrySet()) {
+            CorfuRecord<TableDescriptors, TableMetadata> corfuRecord =
+                    standbyRuntime.getTableRegistry().getRegistryTable().get(entry.getKey());
+            Assert.assertFalse(corfuRecord.getMetadata().getTableOptions().getIsFederated());
```

Comment:
also check that other table entries are replicated correctly, i.e., same as on the Source.

Reply:
That part is covered in `verifyDataOnStandby` method